### PR TITLE
feat: added image building for trivydb

### PR DESCRIPTION
**Pull request by tomhobson**
This adds trivy db container building and pushing through a container registry.

* Runs on main and a cron (7AM)
* Create a changelog using the main jx cli and version images for easier debugging

### CODE REVIEW

File: .github/workflows/jenkins-x/changelog.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+if [ -d "charts/$REPO_NAME" ]; then
+  sed -i -e "s/^version:.*/version: $VERSION/" ./charts/$REPO_NAME/Chart.yaml
+  sed -i -e "s/tag:.*/tag: $VERSION/" ./charts/$REPO_NAME/values.yaml;
+
+  # sed -i -e "s/repository:.*/repository: $DOCKER_REGISTRY\/$DOCKER_REGISTRY_ORG\/$APP_NAME/" ./charts/$REPO_NAME/values.yaml
+else
+  echo no charts
+fi
+
+jx changelog create --verbose --header-file=hack/changelog-header.md --version=$VERSION --rev=$PULL_BASE_SHA --output-markdown=changelog.md --update-release=false
```
Comment (ankitm123) [inline .github/workflows/jenkins-x/changelog.sh]: NIT: new line

File: .github/workflows/trivydb-release.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      with:
+        registry: ghcr.io
+        username: jenkins-x
+        password: ${{ secrets.GHCR_TOKEN }}
+    - name: Build and push trivydb
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./Dockerfile
+        platforms: linux/amd64,linux/arm64
```
Comment (ankitm123) [inline .github/workflows/trivydb-release.yaml]: no darwin?

Comment (tomhobson) [reply .github/workflows/trivydb-release.yaml]: It's only intended to be used within docker, but https://github.com/jenkins-x/jx/blob/main/.github/workflows/jenkins-x-release.yaml#L57 maybe we should look at this one too?

Comment (ankitm123) [reply .github/workflows/trivydb-release.yaml]: so basically our bootjobs wont work on AWS arm based instances (graviton and graviton2) :grimacing: 
https://github.com/jx3-gitops-repositories/jx3-eks-asm/blob/49a3c32254d3c649e07c0cf867e106e8ccc512de/versionStream/git-operator/job.yaml#L31
let's add it here, and then in a subsequent PR we can add it to the boot job. wdyt?

Comment (ankitm123) [reply .github/workflows/trivydb-release.yaml]: darwin/amd64 (I am not sure abt arm variant for darwin, will check)

Comment (tomhobson) [reply .github/workflows/trivydb-release.yaml]: Fixed!

**Pull request merged**